index.html

```diff
@@ -274,6 +295,7 @@
     var EAST      = 3;
     var SOUTHEAST = 4;
     var SOUTHWEST = 5;
+    var sunPosition = NORTHEAST;
```

Comment:
can you move this to line 563 after `var moves`? The variables here are more like constants